arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1660 @@
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/skus": {
+      "get": {
+        "tags": ["GET"],
+        "summary": "Get Valid Skus.",
+        "description": "Get Valid Skus.",
+        "operationId": "IotHubResource_GetValidSkus",
+        "consumes": [],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "The name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "description": "The name of the resource group.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/IotHubSkuDescriptionListResult"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Devices/IotHubs/{resourceName}/eventHubEndpoints/{eventHubEndpointName}/ConsumerGroups": {
+      "get": {
+        "tags": ["GET"],
+        "summary": "Get EventHub Consumer Groups for an IotHub.",
+        "description": "Get EventHub Consumer Groups for an IotHub.",
+        "operationId": "IotHubResource_ListEventHubConsumerGroups",
+        "consumes": [],
+        "produces": ["application/json", "text/json"],
+        "parameters": [{
+          "name": "resourceName",
+          "in": "path",
+          "description": "The name of the resource.",
+          "required": true,
+          "type": "string"
+        }, {
+          "name": "eventHubEndpointName",
+          "in": "path",
+          "description": "The name of the event hub endpoint.",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/subscriptionId"
+        }, {
+          "name": "resourceGroupName",
+          "description": "The name of the resource group.",
+          "in": "path",
+          "required": true,
+          "type": "string"
+        }, {
+          "$ref": "#/parameters/api-version"
+        }],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StringListResult"
```

Comment:
Can we have a better name instead of StringListResult `EventHubConsumerGroupsListResult` or `EventHubConsumerGroups` or `EventHubConsumerGroupList`